src/cc65/pragma.c

```diff
@@ -535,12 +536,15 @@ static void WrappedCallPragma (StrBuf* B)
         goto ExitPoint;
     }
 
-    if (!GetNumber (B, &Val)) {
+    /* Next must be either a numeric value, or "bank" */
+    if (HasStr (B, "bank")) {
+        UseBank = 1;
+    } else if (!GetNumber (B, &Val)) {
```

Comment:
Perhaps it should be changed to "Value or 'bank' required for wrapped-call identifier"?

Reply:
I agree.  But, I would say, "the word 'bank'":
```
"Value or the word 'bank' required for wrapped-call identifier"
```